packages/kbn-data-service/package.json

```
@@ -0,0 +1,6 @@
+{
+  "name": "@kbn/data-service",
+  "private": true,
+  "version": "1.0.0",
+  "license": "SSPL-1.0 OR Elastic License 2.0"
```

```suggestion
  "license": "SSPL-1.0 OR Elastic License 2.0",
  "sideEffects": false
```

As described here: https://github.com/elastic/kibana/pull/158718/files#r1226358320

---

@dej611 seems the `sideEffects` change broke the build due to sass not supporting it?

---

@dej611 Thanks a lot for your suggestions! Trying it now.

---

Tests are passing. Looks good! Thanks again 👏